### DISCUSSION TITLE
Troubleshooting section in docs

### DISCUSSION THREAD

**Discussion by Bond-Addict** [epicmaxco/vuestic-ui#3574] [Ideas]
It would be really nice to have common issues in a trouble shooting sections. For example, I just an issue where I couldn't get a component to work on a new page (after updating). 

The error I was having is I was receiving 'Error: Vuestic cache plugin is not registered!'
After researching on stackoverflow I could not find a solution. Once I put in an issue for the toast component I was using it was pointed out that the import was incorrect. This can happen often because  vscode will automatically create imports which may be wrong, but look correct and still allow the app to build.

Another example I was using 
```
import { useModal } from 'vuestic-ui/web-components';

const { confirm } = useModal();
```
instead of 
```
import { useModal } from 'vuestic-ui';

const { confirm } = useModal();
```

I'm not sure what would be the best format. Maybe to start with a table with common/easy errors?

Something like: 

# Common Errors
| Error Behavior/Error Message | Cause |
|-------|----------|
|Error: Vuestic cache plugin is not registered!| Check vuestic imports. For composables you must import from 'vuestic-ui' and not 'vuestic-ui/web-components' <br/> Example: <br/> ❌ import { useModal } from 'vuestic-ui/web-components' <br/> ✔️  import { useModal } from 'vuestic-ui';|

This would allow us to resolve more of our own issues so the Vuestic team can focus on more important things 😃

**Comment by m0ksem** [comment]
At first, we need to collect the most common issues.

Here is a small list I can find:
- clean cache after upgrading ui version in `node_modules/.vite/deps/` and `.nuxt/`
- usage composables outside of setup function context (not relevant in next release with single app mode introduction).


Idk. Need to fulfil the list at first.

**Reply by Bond-Addict** [reply]
@m0ksem that totally makes sense. I figured at the least it would be a good discussion topic for your team and eventually help the community. Maybe it would be worth creating a poll to see if the community would find value in it. I'm sure long time users could help populate the list with 'gotchas' they've noticed in their time. 

Along with the troubleshoot section, a upgrade guide would be really nice too.
